drivers/media/common/v4l2-tpg/v4l2-tpg-core.c

```diff
@@ -309,6 +309,7 @@ bool tpg_s_fourcc(struct tpg_data *tpg, u32 fourcc)
 	case V4L2_PIX_FMT_NV21M:
 		tpg->buffers = 2;
 		/* fall through */
+	case V4L2_PIX_FMT_SAND8:
```

Comment:
Do we actually need to use the test pattern generator?

Reply:
Probably not - I think I added that line when I was adding SAND cases everywhere I could find NV12 used to make sure I got frames through